.github/workflows/pipeline.yml

```
@@ -0,0 +1,40 @@
+name: 'Pipeline test'
+
+on:
+    workflow_dispatch:
+    pull_request:
+
+permissions:
+  contents: read
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)